service/topology_coordinator.cc

```diff
@@ -752,6 +753,87 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
         case global_topology_request::cleanup:
             co_await start_cleanup_on_dirty_nodes(std::move(guard), true);
             break;
+        case global_topology_request::keyspace_rf_change: {
```

Comment:
nit: why the outer brace is needed?

Reply:
ping

Reply:
I haven't had a chance to go through all the comments yet, I will certainly address all of them.

Reply:
First let me note that the braces around the case statement are not uncommon in the code base. I introduced them not to leak symbols into other cases. Here's the motivation - https://stackoverflow.com/questions/7044574/about-the-braces-in-a-case-statement-in-switch